test/functional/inabox/test-utils.js

```diff
@@ -36,8 +49,11 @@ describes.fakeWin('inabox-utils', {}, env => {
     dispatchEventStub = sandbox.stub();
     initCustomEventStub = sandbox.stub();
     env.win.parent = {postMessage: parentPostMessageStub};
-    env.win.document =
-      {createEvent: () => ({initCustomEvent: initCustomEventStub})};
+    env.win.CustomEvent = (type, eventInit) => {
```

Comment:
why is this needed?

Reply:
This is to mock out [CustomEvent](https://developer.mozilla.org/en-US/docs/Web/API/CustomEvent) on the window. Which is the newer API of of creating events, relative to [createEvent](https://developer.mozilla.org/en-US/docs/Web/API/Document/createEvent).

Since this test now uses a `realWin` instead of a `fakeWin`, it no longer will use the fallback, thus we need to mock out the modern use case.